### ISSUE TITLE
Benchmarking

### ISSUE THREAD

**Issue by jtmoulia** [jtmoulia/neotomex#9]
It's nice to have reports on how performant your grammar is. This should be added as a mix task where you can specify an input file, allow stdin, or perhaps specify a function that would generate the test data as a stream of Strings.

```
mix neotomex.benchmark --grammar=lib/grammar.ex --target=path/to/test_data
```

I'll use [elixir-benchmark](https://github.com/meh/elixir-benchmark) to spawn the tests.

This needs to be done before #6 so that I can understand how it changes Neotomex's performance.
